bug_relationship_graph.php

```diff
@@ -62,20 +62,27 @@
 $f_bug_id		= gpc_get_int( 'bug_id' );
 $f_type			= gpc_get_string( 'graph', 'relation' );
 $f_orientation	= gpc_get_string( 'orientation', config_get( 'relationship_graph_orientation' ) );
+$f_show_summary	= gpc_get_bool( 'summary', false );
```

Comment:
We should have a config option for defaulting the show summary? `relationship_graph_show_summary`.  I would default it to ON.

Reply:
Honestly, I don't think it's really worth adding a new config for this, particularly if you'd like to see the feature moved into a plugin.